packages/documentation/src/layouts/external-layout.js

```diff
@@ -19,7 +18,7 @@ export default class ExternalLayout extends Component {
   }
 }
 
-export const pageQuery = graphql`
+export const pageQuery = `
```

Comment:
Did you try adding the `graphql` function back after you got everything working? 

In one of my failed attempts to fix the build, I removed the `graphql` function, which made one error go away, but still didn't fix the build. 

I wonder if the build would still pass if you reverted the changes to this file. 🤷‍♂️

Reply:
I was tried adding the `graphql` back locally and weirdly the build passes, but one of the steps fail:

```
ERROR #85923  GRAPHQL

There was an error in your GraphQL query:

Cannot query field "fields" on type "MarkdownRemark".

If you don't expect "fields" to exist on the type "MarkdownRemark" it is most likely a typo.
However, if you expect "fields" to exist there are a couple of solutions to common problems:

- If you added a new data source and/or changed something inside gatsby-node.js/gatsby-config.js, please try a restart of your development server
- The field might be accessible in another subfield, please try your query in GraphiQL and use the GraphiQL explorer to see which fields you can query and what shape they have
- You want to optionally use your field "fields" and right now it is not used anywhere. Therefore Gatsby can't infer the type and add it to the GraphQL schema. A quick fix is to add a least one entry with that field ("dummy content")

It is recommended to explicitly type your GraphQL schema if you want to use optional fields. This way you don't have to add the mentioned "dummy content". Visit our docs to learn how you can define the schema for "MarkdownRemark":
https://www.gatsbyjs.org/docs/schema-customization/#creating-type-definitions

File: src/layouts/external-layout.js:26:7

failed extract queries from components - 0.485s
```

With the `graphql` function removed, this step doesn't fail 👀.

Reply:
🤔